pkg/bindings/images/build.go

```diff
@@ -682,8 +682,15 @@ func nTar(excludes []string, sources ...string) (io.ReadCloser, error) {
 						}
 					}
 				}
-				name := filepath.ToSlash(strings.TrimPrefix(path, s+separator))
-
+				var name string
+				if i == 0 {
+					name = filepath.ToSlash(strings.TrimPrefix(path, s+separator))
+				} else {
+					if !d.Type().IsRegular() {
```

Comment:
This is a perfect example of why I ABSOLUTELY LOATHE one char variable names that aren't referenced within 3 or 4 lines.  It took me almost 5 minutes to spot where 'd' was defined.  For the love of all things holy, please add a few characters to this and any other 1 character variables that aren't referenced within a few lines of its definition.